canonical-web-and-design/react-components
# Conditionally load data when available, remove custom chip styling

## Done

- Remove custom chip styling as it is now upstream in Vanilla
- Conditionally load data when available to ensure empty containers do not display

### QA steps

- Open search and filter pattern in sandbox view
- Quickly refresh and open panel and see that headers do not appear until chips appear

## Fixes

Fixes #331, Fixes #332
